backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoice/ManualInvoiceRepository.java

```diff
@@ -179,10 +182,11 @@ private I_C_Invoice saveInvoiceHeader(@NonNull final CreateManualInvoiceRequest
 		invoiceRecord.setC_DocType_ID(createManualInvoiceRequest.getDocTypeId().getRepoId());
 		invoiceRecord.setC_DocTypeTarget_ID(createManualInvoiceRequest.getDocTypeId().getRepoId());
 		invoiceRecord.setPOReference(createManualInvoiceRequest.getPoReference());
-		invoiceRecord.setIsSOTrx(createManualInvoiceRequest.getSoTrx().toBoolean());
+		invoiceRecord.setIsSOTrx(soTrx.toBoolean());
```

Comment:
is there any real change among these lines?

Reply:
Nope, reverted.